packages/esm-patient-medications-app/translations/en.json

```diff
@@ -76,7 +76,6 @@
   "searchResultsMatchesForTerm_one": "{{count}} result for \"{{searchTerm}}\"",
   "searchResultsMatchesForTerm_other": "{{count}} results for \"{{searchTerm}}\"",
   "selectFrequencyErrorMessage": "Please select a frequency",
-  "selectQuantityUnitsErrorMessage": "Please select quantity unit",
```

Comment:
Seems like this message needs to be preserved right?

Reply:
@usamaidrsk Have you seen this?

Reply:
Yeah, though this looks to not exist in the incoming changes.
However I may need to extract new translations

Reply:
It's used [here](https://github.com/openmrs/openmrs-esm-patient-chart/pull/1667/files#diff-03a596ff2777cc23fda9e9c3d119e0ee0ae8cf32a8e79053056ed6d0666b87dcR102). The problem is the parser won't find that without a hint...